solana/ts/sdk/ntt.ts

```diff
@@ -543,16 +545,26 @@ export class SolanaNtt<N extends Network, C extends SolanaChains>
     return BigInt(rl.rateLimit.capacityAtLastTx.toString());
   }
 
-  async getIsExecuted(attestation: Ntt.Attestation): Promise<boolean> {
-    if (!this.getIsApproved(attestation)) return false;
-
-    const { emitterChain } = attestation as WormholeNttTransceiver.VAA;
-    const inboundQueued = await this.getInboundQueuedTransfer(
-      emitterChain,
-      attestation
-    );
+  async getRateLimitDuration(): Promise<bigint> {
+    // The rate limit duration is hardcoded to 24 hours on Solana
+    return BigInt(24 * 60 * 60);
```

Comment:
always?

Reply:
This is hard-coded in the contract and there's no method we can call like we do to get the version